packages/notifications/src/styled/StyledWell.ts

```diff
@@ -22,9 +22,11 @@ export const StyledWell = styled(StyledBase).attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledWellProps>`
-  background-color: ${props => props.isRecessed && getColorV8('neutralHue', 100, props.theme)};
-  color: ${props => getColorV8('neutralHue', 600, props.theme)}
-    ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  background-color: ${p =>
+    p.$isRecessed && getColor({ variable: 'background.recessed', theme: p.theme })};
+  color: ${p => getColor({ variable: 'foreground.subtle', theme: p.theme })};
```

Comment:
Feels like there's enough going on here to warrant a more readable `colorStyles` with good variable declarations.